cli/cmd/vulnerability.go

```diff
@@ -364,13 +394,67 @@ func pollScanStatus(requestID string) error {
 
 		cli.StopProgress()
 		cli.OutputHuman(buildVulnerabilityReportTable(assessment))
+
 		if vulCmdState.Html {
 			return generateVulnAssessmentHTML(assessment)
 		}
+
+		if vulCmdState.FailOnSeverity != "" {
+			vulnContainerFailureThreshold(assessment)
+		}
+
 		return nil
 	}
 }
 
+func vulnFailureThresholdc(assessment api.HostVulnHostAssessment) {
```

Comment:
This needs fixed. The vulnFailureThreshold function is done twice for HostVulnHostAssessment and HostVulnScanPkgManifestResponse. See if this can be reduced to 1 function. If not rename the functions appropriately